examples/browser-webrtc/Cargo.toml

```diff
@@ -21,9 +21,18 @@ rand = "0.8"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 axum = "0.6.19"
-libp2p = { path = "../../libp2p", features = ["ed25519", "macros", "ping", "wasm-bindgen", "tokio"] }
+libp2p = { path = "../../libp2p", features = [
+    "ed25519",
+    "macros",
+    "ping",
+    "wasm-bindgen",
+    "tokio",
+] }
```

Comment:
Please avoid these re-formattings, we don't currently enforce a formatting in CI for toml files so I'd like to keep the noise to a minimum! If you are interested, you could look into a solution to https://github.com/libp2p/rust-libp2p/issues/4043 :)

Reply:
okay. I'll take a look #4043, changes made as per requested.